ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.model.js

```diff
@@ -74,6 +75,14 @@ module.exports = Backbone.AssociatedModel.extend({
       points: [...this.get('points'), point],
     })
   },
+  /*
+   * Appends the given label to the array of labels being tracked.
+   */
```

Comment:
:pencil2: either remove or make more descriptive. As it is , I think the method name is self-documenting